src/app/tabs/tabbed-menu.js

```diff
@@ -1,52 +1,97 @@
 var yo = require('yo-yo')
-var helper = require('../../lib/helper')
+var csjs = require('csjs-inject')
+var remixLib = require('remix-lib')
 
-var css = require('./styles/tabbed-menu-styles')
```

Comment:
yeah putting the css in the same file make sense that's a tiny one, then `tabbed-menu-styles` should be removed